src/renderers/dom/shared/ReactDOMContainerInfo.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactDOMContainerInfo
+ */
+
+'use strict';
+
+var validateDOMNesting = require('validateDOMNesting');
+
+var DOC_NODE_TYPE = 9;
+
+function ReactDOMContainerInfo(node) {
```

Comment:
Nit: Can we make this return a plain (prototype-less) object instead of being a constructor?

Prototypes encourages us to put methods on it that are not statically resolvable.